### DISCUSSION TITLE
Grid: Add Consumption EPS in default view

### DISCUSSION THREAD

**Discussion by ccamejo1** [Security-Onion-Solutions/securityonion#9291] [Ideas]
Currently, one has to expand the individual node in the Grid view to see the Consumption EPS value for any receiver or search node.  It is convenient to have both visible at all times, and has helped identify nodes that may be having issues.

![image](https://user-images.githubusercontent.com/13799970/205684408-1a6b3547-e7ff-4c02-adab-50ccd7f9e6fc.png)

We currently add the field to the page-grid section of the index.html page.  Although we need to re-add it every time there is an update, or we restart the SOC, it's worth the effort.

IMO this would be a great addition to the Grid page.

Keep up the awesome work!!

C2

**Comment by dougburks** [comment]
Have you considered Grafana's searchnode dashboard?

**Comment by ccamejo1** [comment]
Hi Doug,

We use the Grafana searchnode dashboard probably more to look up additional information if we are having an issue.  We like the Grid because it is compact and has all our nodes on one page (currently 24).  Using the Grafana dashboard is a little busy viewing all the searchnodes at once.  Filtring per node is possible on the dashboard, but the Grid has it all right there.  It covers the EPS Consumption for the Receiver nodes as well, all on one page.  
It's not a need but more a convenience.  I just thought I'd throw the idea out there.

Thanks for all you do!

C2

**Comment by dougburks** [comment]
OK, we'll see if we can add this to Grid in the upcoming Security Onion 2.4.

**Comment by dougburks** [comment]
Good news @ccamejo1 ! 

I've added this to our upcoming 2.3.200 release!
https://github.com/Security-Onion-Solutions/securityonion/issues/9359

**Comment by ccamejo1** [comment]
That's awesome!  Thanks, Doug.

C²

On Tue, Dec 13, 2022 at 6:38 AM Doug Burks ***@***.***> wrote:

> Good news @ccamejo1 <https://github.com/ccamejo1> !
>
> I've added this to our upcoming 2.3.200 release!
> #9359
> <https://github.com/Security-Onion-Solutions/securityonion/issues/9359>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/9291#discussioncomment-4388778>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJJEIQJW3PJAADJRGBN6ITWNBN45ANCNFSM6AAAAAASUOPU2M>
> .
> You are receiving this because you were mentioned.Message ID:
> <Security-Onion-Solutions/securityonion/repo-discussions/9291/comments/4388778
> @github.com>
>


-- 
Regards,

C²
Carlos Camejo
C|EH, CHFI, GPEN, GWAPT, SSCP
CompTIA Project+, Linux+

**Comment by dougburks** [comment]
Security Onion 2.3.200 is now available and includes this improvement!
https://blog.securityonion.net/2023/01/security-onion-23200-now-available.html
